lib/constructs/link-processor.ts

```diff
@@ -1,6 +1,6 @@
 /*
 composite construct that sets up all resources
-for links life cycle provisionings
+for account assignments creation/deletion
```

Comment:
is it too annoying to rename the file names at this point, or shall we park this for a later refactor PR?

Reply:
This change needs to be done at multiple places, end point names, DDB table names, API payload samples. Updating the high level comments was my first step towards this change. I would suggest we park this for a later refactor PR as it would beef up an already huge PR